packages/react-cosmos-component-playground/src/components/Tree/index.less

```diff
@@ -38,6 +38,15 @@
   }
 }
 
+.componentChildren {
+  overflow: hidden;
+  transition: max-height 1s ease;
```

Comment:
This doesn't have any effect for the moment, right? Unless you want to give animations another try I say we simplify and rely on good ol `display: none/block`

Reply:
Done